python/lsst/ip/isr/isrTask.py

```diff
@@ -2106,6 +2138,9 @@ def updateVariance(self, ampExposure, amp, overscanImage=None):
             readNoise = afwMath.makeStatistics(overscanImage, afwMath.STDEVCLIP, stats).getValue()
             self.log.info("Calculated empirical read noise for amp %s: %f.",
                           amp.getName(), readNoise)
+        elif self.config.usePtcReadNoise and ptcDataset is not None:
+            readNoise = ptcDataset.noise[amp.getName()]
+            self.log.info("Using read noise from Photon Transfer Curve.")
```

Comment:
Same logic as above - if people are asking for that, I think it should be a raise if they're not present, rather than just quietly proceeding.